src/managers/GuildMemberManager.js

```diff
@@ -238,21 +246,25 @@ class GuildMemberManager extends BaseManager {
           presences,
           user_ids,
           query,
+          nonce: nonce.substring(0, 32),
```

Comment:
Truncating the nonce here will cause the comparison below to fail as it is not being truncated there too.

I personally think it might be better to throw an error if the nonce is too long instead of silently truncating it.